### PR TITLE
Switch to a compliant base image

### PR DESCRIPTION
**Pull request by aermakov-zalando**
I want to stop getting the emails. :(

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.opensource.zalan.do/stups/alpine:latest
+FROM registry.opensource.zalan.do/library/alpine-3.12:3.12-20201117
```
Comment (mikkeloscar) [inline Dockerfile:1]: can't we use `:latest` here? so we don't have to manually update in the future?

Comment (aermakov-zalando) [reply Dockerfile]: Fixed!

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**